plugins/proofpoint_tap/help.md

```diff
@@ -74,225 +74,165 @@ Example input:
 |threatStatus|string|all|True|The threat statuses which will be returned in the data|["active", "cleared", "falsePositive", "all"]|active|None|None|
+|threatType|string|all|True|The threat type which will be returned in the data|["url", "attachment", "messageText", "all"]|url|None|None|
 |timeEnd|date|None|False|The end of the data retrieval period as ISO8601-formatted date e.g 2021-04-20T22:00:00Z. If left empty, it will be calculated from the 'time_start' parameter. If the 'time_start' parameter is empty, data from one hour before the current API server time will be returned. The minimum time range is thirty seconds. The maximum time range is one hour|None|2021-04-20T22:00:00+00:00|None|None|
 |timeStart|date|None|False|The start of the data retrieval period as ISO8601-formatted date e.g 2021-04-20T21:00:00Z. If left empty, it will be calculated from the 'time_end' parameter. If the 'time_end' parameter is empty, data from one hour before the current API server time will be returned. The minimum time range is thirty seconds. The maximum time range is one hour|None|2021-04-20T21:00:00+00:00|None|None|
-|url|string|None|False|The URL for which the results will be returned. Returns all results if left empty|None|https://example.com|None|None|
-
+  
 Example input:
 
 ```
 {
-  "threatStatus": "active",
+  "subject": "A phishy email",
+  "threatStatus": "all",
+  "threatType": "all",
   "timeEnd": "2021-04-20T22:00:00+00:00",
-  "timeStart": "2021-04-20T21:00:00+00:00",
-  "url": "https://example.com"
+  "timeStart": "2021-04-20T21:00:00+00:00"
 }
 ```
 
 ##### Output
 
 |Name|Type|Required|Description|Example|
-|----|----|--------|-----------|-------|
-|results|permittedClicks|True|The results containing permitted clicks|{}|
-
+| :--- | :--- | :--- | :--- | :--- |
+|results|blockedMessages|True|The results containing blocked messages|{}|
+  
 Example output:
 
 ```
 {
-  "results": {
-    "clicksPermitted": [
-      {
-        "GUID": "X7sh5TwRxBZOAXb-d8ESyugsIdtfv3u",
-        "classification": "malware",
-        "clickIP": "198.51.100.1",
-        "clickTime": "2021-04-20T21:08:13.000Z",
-        "id": "9de5069c-5afe-602b-2ea0-a04b66beb2c0",
-        "messageID": "<user@example.com>",
-        "recipient": "user@example.com",
-        "sender": "user@example.com",
-        "senderIP": "198.51.100.1",
-        "threatID": "f1f23718b35b8db3db005cd498ff0812e53fe994537567ff0a...",
-        "threatStatus": "active",
-        "threatTime": "2021-04-20T21:08:38.000Z",
-        "threatURL": "https://example.com",
-        "url": "https://example.com",
-        "userAgent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/89.0.4389.128 Safari/537.36"
-      }
-    ],
-    "queryEndTime": "2021-04-21T13:00:00Z"
-  }
+  "results": {}
```

Comment:
There looks to be a handful of examples that are not added to the spec and are coming in as empty to help md

Reply:
These include the outputs for the following 

- Get All Threats
- Get Blocked Clicks
- Get Blocked Messages
- Get Delivered Threats
- Get Permitted Clicks
- Get Top Clickers
- Parse Alert
- URL Decode